src/benchmarks/micro/corefx/System.Drawing/Perf_Image_Load.cs

```diff
@@ -50,6 +46,28 @@ public void Image_FromStream_NoValidation(ImageTestData format)
             }
         }
 
+        private static ImageTestData[] CreateTestCases()
+        {
+            try
+            {
+                return new [] 
+                {
+                    new ImageTestData(ImageFormat.Bmp),
+                    new ImageTestData(ImageFormat.Jpeg),
+                    new ImageTestData(ImageFormat.Png),
+                    new ImageTestData(ImageFormat.Gif)
+                };
+            }
+            catch (Exception) when (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.WriteLine("libgdiplus is missing, you can install it by running 'apt-get install libgdiplus'");
```

Comment:
is it worth it to call out `sudo` here?

Reply:
I do not think so, the user might be already sudo or have permissions.